### PR TITLE
Migration: wwf updates

### PR DESCRIPTION
**Pull request by guardrex**
Ready for a look :eyes:

Notes
* MSDN converted the link to `System.WorkflowModel.Activities.Activity` (depreciated) to just "Activity" (as an unresolved link), and I did the same here.
* `System.Activities.CodeActivity.OnGetConstraints()` is gone. Looks like the sample app for basic validation is updated, but the docs at *basic-validation.md* and *contraint-types.md* weren't updated. I converted to inline code and noted in the Migration Project that these sample topics need more work. Ref: https://social.msdn.microsoft.com/Forums/vstudio/en-US/319acbd4-fe90-4d68-a0df-1f8056a924c8/where-has-gone-the-ongetconstraints-in-beta-2?forum=wfprerelease
* MIA unless gone: <code>System.Activities.Statements.InvokeMethod\`1</code> (generic version; https://msdn.microsoft.com/library/dd647677.aspx) and <code>System.Activities.Statements.InvokeMethod\`1.Result</code> (https://msdn.microsoft.com/library/dd987724.aspx). I used the MSDN links and noted in the project to investigate.

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/samples/basic-validation.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This sample consists of an activity, `CreateProduct`, which validates that its `
   
 -   Create a constraint (`PriceGreaterThanCost`). This is where all the validation logic resides.  
   
--   Override <xref:System.Activities.CodeActivity%601.OnGetConstraints%2A> and add the constraint (`PriceGreaterThanCost`) to the constraints <xref:System.Collections.IList>.  
+-   Override `System.Activities.CodeActivity.OnGetConstraints()` and add the constraint (`PriceGreaterThanCost`) to the constraints <xref:System.Collections.IList>.
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/samples/basic-validation.md]: if this is gone, shouldn't we remove it? or are we saying is a more substantial work? if so, let's open an issue.

**Review by mairaw — APPROVED** [review]
Looks good. Just had a question about the API that was removed from the product.

File: docs/framework/windows-workflow-foundation/samples/basic-validation.md (hunk 1 above)
Comment (GuardRex) [inline docs/framework/windows-workflow-foundation/samples/basic-validation.md]: AFAICT they updated the samples but not the docs. Yes, it needs more extensive work. This is noted in the Migration project for further action. I'll open an issue on Monday and exchange the note in the Migration Project with the issue I open.

Comment (mairaw) [inline docs/framework/windows-workflow-foundation/samples/basic-validation.md]: If we have it as a note, it's ok as well... as long as we're tracking this somewhere.

**Pull request merged**